MapleServer2/Packets/CharacterListPacket.cs

```diff
@@ -99,8 +99,8 @@ public static void WriteCharacter(Player player, PacketWriter pWriter)
             pWriter.WriteInt();
             pWriter.WriteShort(player.Levels.Level);
             pWriter.WriteShort();
-            pWriter.WriteInt(player.JobGroupId);
-            pWriter.WriteInt(player.JobId);
+            pWriter.WriteInt((int) player.Job);
```

Comment:
Can we just change `WriteMode` to `WriteEnum` or make a new method `WriteEnum` that will convert these automatically?